config/contracts.js

```diff
@@ -101,7 +103,101 @@ module.exports = {
         }
       ]
     },
-    "afterDeploy": ["SNT.methods.generateTokens('$accounts[0]', '100000000000000000000').send()"]
+    afterDeploy: async (deps) => {
+      const addresses = await deps.web3.eth.getAccounts();
+      const main = addresses[0];
+      const sntToken = 100;
+      const balance = await deps.contracts.SNT.methods.balanceOf(main).call();
+      if (balance === '0') {
```

Comment:
Can you reverse the condition and `return`? Gonna make it easier to read.

Reply:
definitively, guard close :100: